src/.eslintrc.js

```diff
@@ -24,6 +24,20 @@ module.exports = {
             files: ['*.mjs', '*.ts', '*.d.ts'],
             parserOptions: { sourceType: 'module' },
         },
+        {
+            files: [
+                '**/*.test.js', '**/*.test.ts',
+            ],
+            env: { jest: true },
+            plugins: ['jest'],
+            rules: {
+                'jest/no-disabled-tests': 'warn',
+                'jest/no-focused-tests': 'error',
+                'jest/no-identical-title': 'error',
+                'jest/prefer-to-have-length': 'warn',
+                'jest/valid-expect': 'error',
+            },
+        },
```

Comment:
So what can be better here is to defer to the list of "recommended" rules from ESLint:

```json
{
  "extends": ["plugin:jest/recommended"]
}
```

Personally I would also suggest the auto-fixable style rules to enforce consistency in our tests: https://www.npmjs.com/package/eslint-plugin-jest#rules

...or we also just additionally include those:

```json
{
  "extends": ["plugin:jest/style"]
}
```